config/cdConfig.js

```diff
@@ -95,7 +95,7 @@ module.exports = {
         // '--quiet'
       ],
       timeout: 1000,
-      processes: 2,
+      processes: Number(process.env.CRAWLER_SCANCODE_PARALLELISM) || 2,
```

Comment:
To allow this to be set using the normal configuration approach or from an env var, follow the pattern in... https://github.com/clearlydefined/crawler/blob/0738316eaf5a9ef80f96cdaaeae38e2695688974/ghcrawler/bin/www.js#L14

```suggestion
      processes: Number(config.get(CRAWLER_SCANCODE_PARALLELISM) || process.env.CRAWLER_SCANCODE_PARALLELISM) || 2,
```